capstone-backend/src/test/java/com/google/sps/ClubTest.java

```diff
@@ -0,0 +1,214 @@
+    Assert.assertEquals(list.get(1), new Long(MEMBER_TWO));
+  }
+
+  @Test
+  public void testUninvite() {
+    club.invite(MEMBER_ONE);
+    club.invite(MEMBER_TWO);
+    club.uninvite(MEMBER_TWO);
+    Assert.assertEquals(club.getInviteIDs().size(), 1);
+    List<Long> list = new ArrayList<>(club.getInviteIDs());
+    Assert.assertEquals(list.get(0), new Long(MEMBER_ONE));
+  }
+
+  @Test
+  public void testUninviteList() {
+    club.invite(new ArrayList<>(Arrays.asList(MEMBER_ONE, MEMBER_TWO, MEMBER_THREE)));
+    club.uninvite(new ArrayList<>(Arrays.asList(MEMBER_TWO, MEMBER_THREE)));
+    Assert.assertEquals(club.getInviteIDs().size(), 1);
+    List<Long> list = new ArrayList<>(club.getInviteIDs());
+    Assert.assertEquals(list.get(0), new Long(MEMBER_ONE));
+  }
+
+  @Test
+  public void testAddMemberInvited() {
+    club.invite(MEMBER_ONE);
+    club.invite(MEMBER_TWO);
+    boolean b = club.addMember(MEMBER_ONE);
+
+    Assert.assertEquals(b, true);
+    Assert.assertEquals(club.getInviteIDs().size(), 1);
+    Assert.assertEquals(club.getMemberIDs().size(), 1);
+
+    List<Long> list = new ArrayList<>(club.getInviteIDs());
+    Assert.assertEquals(list.get(0), new Long(MEMBER_TWO));
+
+    list = new ArrayList<>(club.getMemberIDs());
+    Assert.assertEquals(list.get(0), new Long(MEMBER_ONE));
+  }
+
+  @Test
+  public void testAddMemberNotInvited() {
+    club.invite(MEMBER_ONE);
+    club.invite(MEMBER_TWO);
+    boolean b = club.addMember(MEMBER_THREE);
+
+    Assert.assertEquals(b, false);
+    Assert.assertEquals(club.getInviteIDs().size(), 2);
+    Assert.assertEquals(club.getMemberIDs().size(), 0);
+
+    List<Long> list = new ArrayList<>(club.getInviteIDs());
+    Assert.assertEquals(list.get(0), new Long(MEMBER_ONE));
+    Assert.assertEquals(list.get(1), new Long(MEMBER_TWO));
+  }
+  
+  @Test
+  public void removeMember() {
+    club.invite(new ArrayList<>(Arrays.asList(MEMBER_ONE, MEMBER_TWO, MEMBER_THREE)));
+    club.addMember(MEMBER_ONE);
+    club.addMember(MEMBER_TWO);
+    club.addMember(MEMBER_THREE);
+    Assert.assertEquals(club.getMemberIDs().size(), 3);
+    club.removeMember(MEMBER_TWO);
+    Assert.assertEquals(club.getMemberIDs().size(), 2);
+    List<Long> list = new ArrayList<>(club.getMemberIDs());
+    Assert.assertEquals(list.get(0), new Long(MEMBER_ONE));
+    Assert.assertEquals(list.get(1), new Long(MEMBER_THREE));
+  }
+
+  @Test
+  public void removeMembers() {
+    club.invite(new ArrayList<>(Arrays.asList(MEMBER_ONE, MEMBER_TWO, MEMBER_THREE)));
+    club.addMember(MEMBER_ONE);
+    club.addMember(MEMBER_TWO);
+    club.addMember(MEMBER_THREE);
+    Assert.assertEquals(club.getMemberIDs().size(), 3);
+    club.removeMembers(new ArrayList(Arrays.asList(MEMBER_ONE, MEMBER_TWO)));
+    Assert.assertEquals(club.getMemberIDs().size(), 1);
+    List<Long> list = new ArrayList<>(club.getMemberIDs());
+    Assert.assertEquals(list.get(0), new Long(MEMBER_THREE));
+  }
```

Comment:
I feel like these should have some newlines in in it to break up the chunks of tests, like a line after 187 and 201 to separate your arranging step?

Reply:
good idea, will fix.